db/controllers/categories.controllers.js

```diff
@@ -1,4 +1,10 @@
-const { fetchCategories } = require("../models/categories.models")
+const { fetchCategories, fetchEndpoints } = require("../models/categories.models")
+
+exports.getEndpoints= (request, response) => {
+    fetchEndpoints().then((endpoints) => {
+        response.status(200).send({ endpoints: endpoints })
```

Comment:
Great use of destructuring with a custom key here in order to keep the code super clean! 🚀 There is no need to invoke the model for this, as we are not querying the database.